pkg/flare/archive.go

```diff
@@ -295,11 +295,14 @@ func writeStatusFile(tempDir, hostname string, data []byte) error {
 
 func addParentPerms(dirPath string, permsInfos permissionsInfos) {
 	parent := filepath.Dir(dirPath)
+
+	// Don't enter loop on root, we don't want to add anything if root is passed as `dirPath`
 	for parent != "." {
```

Comment:
Can you add a comment explaining why you need `parent != "."` as it is not obvious IMO.

Reply:
Will do. filepath.Dir returns “.” On an empty path, that’s why.